packages/twenty-server/src/engine/core-modules/billing/utils/billing-phase.utils.ts

```diff
@@ -0,0 +1,17 @@
+/* @license Enterprise */
+
+export function normalizePriceRef(
+  p: string | { id: string } | null | undefined,
+): string | undefined {
+  if (!p) return undefined;
+
+  return typeof p === 'string' ? p : p.id;
+}
+
+export function ensureFutureStartDate(
```

Comment:
Nit: one file per util (make sure it's a precise name like something that contains billing if it's not a generic util)